### PR TITLE
First Commit - Evaluation

### PR DESCRIPTION
**Pull request by Karthi-DStech**
Greetings, 

I've added scripts for evaluation and it works very well. I've tested the code multiple times and it works very well. Kindly review the code and do the needful. 

Also when trying to run the main code try the follwing below: 

- python -m evaluation.eval_main 

or

- python eval_main.py

### CODE REVIEW

File: Carrot-Assistant/evaluation/eval_main.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+from datetime import datetime
```
Comment (kuraisle) [inline Carrot-Assistant/evaluation/eval_main.py:3]: unused import

**Comment by kuraisle** [comment]
So as I understand it:
- entry point is eval_main.py
- EvaluationOptions are loaded
    - I'm not sure what the point of adding to the sys path is, could you illuminate me?
- EvaluationLogger is initialized
    - This is fine, but not a priority
- ConceptCalculation is created and run
    - tries to process the dataset (take expected values and inputs)
    - queries OMOP
    - LLM
    - OMOP with LLM names
- uses make_evaluation to run the metrics
    - This takes metrics from metrics.py
- saves the log, which has been updating the whole time

Is there a reason you didn't use the ABCs in eval.py? This works for the one pipeline, but I had envisioned a version that was more able to test arbitrary pipelines with some subset of the compatible metrics

**Comment by Karthi-DStech** [comment]
Hi James,

I opted not to use ABC initially due to the specific requirements we had at the time. I believed, a simpler implementation would better serve our immediate needs. However, I understand the benefits of using ABCs and I'm willing to review the ABC implementation. I will research on how to exactly implementation of ABC refactoring our current approach.

Regarding the issues with running eval_main.py, I encountered several errors that prevented the script from executing correctly. To resolve these, I added the script's directory to the system path, which treats the scripts as part of the directory structure, allowing them to be executed without errors. This adjustment has worked well and I'm open for your suggestions as well. 

Let me know for further changes James.

File: Carrot-Assistant/evaluation/eval_main.py (hunk 1 above)
Comment (Karthi-DStech) [reply Carrot-Assistant/evaluation/eval_main.py:3]: Ok I'll remove it.

**Pull request closed without merging**